### PR TITLE
♻️ Removed padding for headings inside grey boxes

### PR DESCRIPTION
**Pull request by jeoffreyfischer**
**Fix**
I have removed **top** padding from headings if they are the **1st** element inside a greybox.
I have also removed **bottom** padding from headings if they are the **last** element inside a greybox.

Relates to #1397 

**Screenshot**
image
**Figure: Margin is removed for headings at the top or bottom of a greybox**

### CODE REVIEW

**Review by bradystroud — DISMISSED** [review]

**Comment by jeoffreyfischer** [comment]
✅ Fixed for
- greybox
- info

❌ Not fixed for
- info
- highlight
- china
- codeauditor
- todo

As per my conversation with @tiagov8, we have no idea why the second list is not fixed (info, highlight...). 
We will deploy the changes to Staging to check whether they work there.
Note: the main use case was spacing in greybox, so ut's not the end of the world if the other boxes heading spacing is not fixed as they are rarely used.

File: package.json [hunk 1]
```diff
@@ -9,8 +9,9 @@
     "build:favicons": "node scripts/favicons",
     "build": "yarn run build:dir && yarn run build:favicons && gatsby build --prefix-paths --log-pages && yarn updateRelativePath",
     "build:incremental": "gatsby build --prefix-paths --log-pages && yarn updateRelativePath",
+    "cleandev": "gatsby clean && gatsby develop",
     "devbuild": "gatsby clean && yarn run build:dir && yarn run build:favicons && gatsby build",
-    "develop": "gatsby develop",
+    "dev": "gatsby develop",
```
Comment (bradystroud) [inline package.json:14]: This is a good change, but check the README doesnt mention the old command

**Review by bradystroud — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (jeoffreyfischer) [reply package.json:14]: Sorry I missed that comment before merhing.
You're right, it does mention "yarn develop" 
I will update the ReadMe in a new PR 👍

**Pull request merged**